run-example.sh

```diff
@@ -58,13 +59,13 @@ trap finish EXIT
 docker-compose -f ./dev/docker-compose.yaml -f ./examples/docker-compose.yaml up -d
 
 # instrument and run HTTP server app in background
-export OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS="Examples.AspNetCoreMvc.OtelSdkPlugin, Examples.AspNetCoreMvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null:Examples.Vendor.Distro.Plugin, Examples.Vendor.Distro, Version=0.0.1.0, Culture=neutral, PublicKeyToken=null"
-ENABLE_PROFILING=${enableProfiling} OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS="AspNet,MongoDb,SqlClient" OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES="MyCompany.MyProduct.MyLibrary" OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS="NetRuntime" OTEL_SERVICE_NAME="aspnet-server" OTEL_TRACES_EXPORTER=${exporter} ./dev/instrument.sh ASPNETCORE_URLS="http://127.0.0.1:8080/" dotnet ./examples/AspNetCoreMvc/bin/${configuration}/${aspNetAppTargetFramework}/Examples.AspNetCoreMvc.dll &
-unset OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS
+export {OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS,OTEL_DOTNET_AUTO_METRICS_PLUGINS}="Examples.AspNetCoreMvc.OtelSdkPlugin, Examples.AspNetCoreMvc, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null:Examples.Vendor.Distro.Plugin, Examples.Vendor.Distro, Version=0.0.1.0, Culture=neutral, PublicKeyToken=null"
+ENABLE_PROFILING=${enableProfiling} OTEL_DOTNET_AUTO_TRACES_ENABLED_INSTRUMENTATIONS="AspNet,MongoDb,SqlClient" OTEL_DOTNET_AUTO_METRICS_ADDITIONAL_SOURCES="MyCompany.MyProduct.MyLibrary" OTEL_DOTNET_AUTO_METRICS_ENABLED_INSTRUMENTATIONS="NetRuntime" OTEL_SERVICE_NAME="aspnet-server" OTEL_TRACES_EXPORTER=${tracesexporter} OTEL_METRICS_EXPORTER=${metricsexporter} ./dev/instrument.sh ASPNETCORE_URLS="http://127.0.0.1:8080/" dotnet ./examples/AspNetCoreMvc/bin/${configuration}/${aspNetAppTargetFramework}/Examples.AspNetCoreMvc.dll &
+unset OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS OTEL_DOTNET_AUTO_METRICS_PLUGINS
```

Comment:
This makes me think that we should rename `OTEL_DOTNET_AUTO_TRACES_INSTRUMENTATION_PLUGINS` to `OTEL_DOTNET_AUTO_TRACES_PLUGINS`. However, that it is unrelated to this change.

Reply:
Created #835 to rename after merging this PR.